### PR TITLE
fixed findbugs hz-cloud

### PR DESCRIPTION
**Pull request by bilalyasar**

### CODE REVIEW

**Comment by pveentjer** [comment]
```
15:32:18 [INFO] Finished at: Fri May 06 12:32:18 UTC 2016
15:32:18 [INFO] Final Memory: 109M/1715M
15:32:18 [INFO] ------------------------------------------------------------------------
15:32:18 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5.1:testCompile (default-testCompile) on project hazelcast-cloud: Compilation failure: Compilation failure:
15:32:18 [ERROR] /appdisk/jenkins/jenkins/workspace/new-lab-fast-pr/hazelcast-cloud/src/test/java/com/hazelcast/aws/DescribeInstancesTest.java:[34,8] unreported exception java.io.IOException; must be caught or declared to be thrown
15:32:18 [ERROR] /appdisk/jenkins/jenkins/workspace/new-lab-fast-pr/hazelcast-cloud/src/test/java/com/hazelcast/aws/DescribeInstancesTest.java:[39,8] unreported exception java.io.IOException; must be caught or declared to be thrown
15:32:18 [ERROR] /appdisk/jenkins/jenkins/workspace/new-lab-fast-pr/hazelcast-cloud/src/test/java/com/hazelcast/aws/DescribeInstancesTest.java:[47,8] unreported exception java.io.IOException; must be caught or declared to be thrown
```

File: hazelcast-cloud/src/main/java/com/hazelcast/aws/impl/DescribeInstances.java [hunk 1]
```diff
@@ -70,19 +70,29 @@ public DescribeInstances(AwsConfig awsConfig, String endpoint) {
...
             String query = "latest/meta-data/iam/security-credentials/";
             URL url;
             url = new URL("http", IAM_ROLE_ENDPOINT, query);
-            InputStreamReader is = new InputStreamReader(url.openStream(), "UTF-8");
-            BufferedReader reader = new BufferedReader(is);
+            is = new InputStreamReader(url.openStream(), "UTF-8");
+            reader = new BufferedReader(is);
             awsConfig.setIamRole(reader.readLine());
-        } catch (Exception e) {
+            is.close();
```
Comment (mesutcelik) [inline hazelcast-cloud/src/main/java/com/hazelcast/aws/impl/DescribeInstances.java]: Are you closing the stream both here and in `finally` block?

Comment (bilalyasar) [inline hazelcast-cloud/src/main/java/com/hazelcast/aws/impl/DescribeInstances.java]: @mesutcelik thanks, updated the code.

**Comment by hasancelik** [comment]
👍

**Pull request merged**